models/cryptopunks/ethereum/cryptopunks_ethereum_current_bids.sql

```diff
@@ -28,15 +28,15 @@ with combined_events_table as (
 
         select  'Transfer' as event_type
                 , cast(NULL as varchar(5)) as bidder
-                , from as transfer_from
+                , t.from as transfer_from
```

Comment:
This line may cause the test error mistakenly.
https://github.com/dbt-checkpoint/dbt-checkpoint/blob/main/dbt_checkpoint/check_script_has_no_table_name.py#L63